.github/workflows/build-docs.yml

```diff
@@ -25,4 +25,4 @@ jobs:
       uses: maxheld83/ghpages@v0.2.1
       env:
         BUILD_DIR = "docs/"
-        secrets = ["GH_PAT"]
+      secrets: ${{ secrets.GH_PAT }}
```

Comment:
This may need to be indented (part of env:) - this is based on https://github.com/maxheld83/ghpages/blob/master/README.md where it isn't in env, but i think that may be wrong

Reply:
changed